single_chart_migration/migration.sh

```diff
@@ -191,9 +192,59 @@ do
             shift
             shift
             ;;
+        --mongo-trail-migration)
+            echo "mongotrailmigration set true"
+            mongotrailmigration=true
+            shift
+            shift
+            ;;
+
     esac
 done
 
+if [ "$mongotrailmigration" == true ]; then
+    echo "mongotrailmigration case"
+    # Check for the presence of required files.
+    if [ ! -f "mongo.yaml" ]; then
+        echo "mongo.yaml is missing. Please download it. (curl https://raw.githubusercontent.com/portworx/helm/master/single_chart_migration/mongo.yaml -o mongo.yaml)"
+    fi
+    if [ ! -f "migrationctl" ]; then
+        echo "migrationctl is missing. Please download it. (curl https://raw.githubusercontent.com/portworx/helm/master/single_chart_migration/migrationctl -o migrationctl)"
+    fi
+    # Get the storage class from the etcd PVC and use it for mongo PVC.
+    sc=`$kubectl_cmd get pvc pxc-etcd-data-pxc-backup-etcd-0 -n $namespace -o yaml | grep storageClassName | awk '{print $2}'`
+    echo "using storage class for mongo $sc"
+    # update the storageclass in mongo.yaml
+    sed -i 's|'storageClassName:.*'|'"storageClassName: $sc"'|g' mongo.yaml
+    # Install mongoDB
+    $kubectl_cmd apply -f ./mongo.yaml -n $namespace
+    echo "Waiting for mongoDB to be in running state"
+    for i in $(seq 1 100) ; do
+        replicas=$(kubectl get statefulset -n px-backup pxc-backup-mongodb-trail -o json | jq ".status.readyReplicas")
+        if [ "$replicas" == "3" ]; then
+            deploy=true
+            break
+        else
+            echo "mongodb datastore is not ready yet"
+            sleep 10
+        fi
+    done
+    $kubectl_cmd scale --replicas=0 deploy/px-backup -n $namespace
+    sleep 10
+    $kubectl_cmd cp ./migrationctl -n $namespace pxc-backup-mongodb-trail-0:/tmp
+    $kubectl_cmd exec -it pxc-backup-mongodb-trail-0 -n $namespace -- /tmp/migrationctl
+    # Delete all the resources of the trail mongoDB deployment.
+    $kubectl_cmd delete sts pxc-backup-mongodb-trail -n $namespace
```

Comment:
we can do "$kubectl_cmd delete -f ./mongo.yaml -n $namespace" , individually deleting the resources is fine too.